creator.html

```diff
@@ -324,24 +318,12 @@ <h5 class="feature-title font-bold mb-1">Frequent Updates</h5>
                             <i class="fa fa-university fa-2x mb-1 deep-purple-text" aria-hidden="true"></i>
                         </div>
                         <div class="col-10 mb-2">
-                            <h5 class="feature-title font-bold mb-1">FREE Forever</h5>
+                            <h5 class="feature-title font-bold mb-1">Free. Forever.</h5>
                             <p class="grey-text mt-2">We decided that Kodular was going to be FREE forever, without any premium plan or compulsory fees for usage.</p>
                         </div>
                     </div>
                 </div>
             </div>
         </section>
-
```

Comment:
This block shouldn't be removed

Reply:
I removed it because it might be a bit confusing for new visitors. Because everywhere else on the website you guys are claiming that to create apps with Kodular no coding skills are required, but then here you are confusing your visitors by adding this section, "How is it **coded**?" Only those who are familiar with AI and other distros know that the _it_ here is referring to the creator but the new visitors don't.

Reply:
Change it to "How has been Kodular Creator made with?"

Reply:
Done.

Reply:
@barreeeiroo